src/Files.App/ViewModels/SettingsViewModels/FoldersViewModel.cs

```diff
@@ -10,6 +10,9 @@ public class FoldersViewModel : ObservableObject
 	{
 		private IUserSettingsService UserSettingsService { get; } = Ioc.Default.GetRequiredService<IUserSettingsService>();
 
+		private readonly int FileTagSortingIndex = 5;
+		private readonly int FileTagGroupingIndex = 6;
```

Comment:
What are these values for?

Reply:
They are used to select the correct option in the comboboxes. Sync status, original path and delete date are hidden because they aren't available for regular folders, so if Tags is set as default the casting would return an invalid index. At first I swapped FileTag and SyncStatus but that enum is referenced from many places.

Reply:
Can you add a comment to the code?

Reply:
Yes, I'll do it